tests/history.rs

```diff
@@ -70,7 +70,7 @@ mod test {
 
         {
             let mut resp = reqwest::get("http://localhost:8000/api/data/feature/1/history").unwrap();
-            assert_eq!(resp.text().unwrap(), r#"[{"feat":{"action":"create","geometry":{"coordinates":[0.0,0.0],"type":"Point"},"id":1,"message":"Creating a Point","properties":{"number":"123"},"type":"Feature"},"id":1,"uid":1,"username":"ingalls"}]"#);
+            assert_eq!(resp.text().unwrap(), r#"[{"feat":{"action":"create","geometry":{"coordinates":[0,0],"type":"Point"},"id":1,"key":null,"properties":{"number":"123"},"type":"Feature","version":1},"id":1,"uid":1,"username":"ingalls"}]"#);
```

Comment:
Some important between the previous feature history endpoint and the current:
- coordinates are stored in a `GEOMETRY` type rather than as pure JSON and as a result are simplified (e.g. `0.0` -> `0`)
- `key` values are always returned, even if they're not included in the original request
- the `version` returned reflects the actual feature version, not the version requested. For example, when a feature is created, it has a version of 1
- the `message` property which was only passed as a property on the feature when a single feature was created/deleted/modified/restored, is recorded in the `deltas` table but never returned from the history endpoint. Previously, it was included in the response when a single feature was created/deleted/modified/restored but not passed if multiple features were changed